test/jdk/javax/imageio/CachePremissionsTest/CachePermissionsTest.java

```diff
@@ -73,17 +48,6 @@ public static void main(String[] args) {
 
         System.out.println("java.io.tmpdir is " + System.getProperty("java.io.tmpdir"));
 
-        if (args.length > 1) {
```

Comment:
The isFileCacheExpected logic does not make sense. The test sets set to use the cache but then reads whether to expect it based on the args[0]. If that were set to false  the test will fail. So why is it there ?

Also the messing around with exceptions at the end of the test is pointless

Reply:
@prrace I might have missed removing this check which was in the original test. The latest update to this test has two run tags but it fails when isFileCacheExpected is set to true.

Did you mean to keep only one run tag? https://github.com/openjdk/jdk-sandbox/commit/1bf77a393c5756bca65760402077617d37be72d2

I'll be rename the test as suggested when I update this test next.

Reply:
No changes required for this test, it works. The test was failing due to IDE config issue of tmp dir.